doc/classes/Node.xml

```diff
@@ -255,7 +255,7 @@
 			<return type="Node" />
 			<param index="0" name="flags" type="int" default="15" />
 			<description>
-				Duplicates the node, returning a new node with all of its properties, signals and groups copied from the original. The behavior can be tweaked through the [param flags] (see [enum DuplicateFlags]).
+				Duplicates the node, returning a new node with all of its properties, signals, groups and children copied from the original. The behavior can be tweaked through the [param flags] (see [enum DuplicateFlags]).
```

Comment:
It looks nitpicky but we actually use the [Oxford comma](https://docs.godotengine.org/en/stable/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything), so we may as well fix this in one sweep, too.
```suggestion
				Duplicates the node, returning a new node with all of its properties, signals, groups, and children copied from the original. The behavior can be tweaked through the [param flags] (see [enum DuplicateFlags]).
```